src/common/api/index.ts

```diff
@@ -0,0 +1,5 @@
+export namespace API {
+  export function getClient() {
+    return true;
+  }
+}
```

Comment:
Why do we need namespace? There's much talk of it being a deprecated feature https://michelenasti.com/2019/01/23/is-typescript-namespace-feature-deprecated.html

Reply:
or rather, not deprecated, but generally best avoided being usd